services/venice-server/src/main/java/com/linkedin/venice/listener/ListenerService.java

```diff
@@ -162,7 +163,14 @@ public ListenerService(
         .option(ChannelOption.SO_BACKLOG, nettyBacklogSize)
         .childOption(ChannelOption.SO_KEEPALIVE, true)
         .option(ChannelOption.SO_REUSEADDR, true)
-        .childOption(ChannelOption.TCP_NODELAY, true);
+        .childOption(ChannelOption.TCP_NODELAY, true)
+        // A higher buffer watermark will allow more data write for a given channel and it would be useful for H2,
+        // as H2 works with a smaller number of connections.
+        .childOption(
+            ChannelOption.WRITE_BUFFER_WATER_MARK,
```

Comment:
Do you know what is the default value for this channel option?

I'm fine with changing Venice's default to 1 MB, but it would be good if the commit message included details about what the default was previously, in case we need to change it back...

Also, the commit message and code comment above mention that this change is useful for HTTP/2, but will it also affect HTTP/1?

Reply:
The default value is 64KB.
I can mentioned that in the commit message.

This will affect HTTP/1 as well, and it can be more useful for H2.
Recently, we noticed a weird behavior as sometimes flush latency can go up high and go down after some period, so I would like to test whether this particular tuning is helpful or not.

Reply:
Maybe I should keep the default value and tune the config explicitly in the experimental env and if there is a need, we can change the default later on.